jbehave-support-core/pom.xml

```diff
@@ -217,6 +217,14 @@
             <groupId>org.springframework</groupId>
             <artifactId>spring-jdbc</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-aspects</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.retry</groupId>
+            <artifactId>spring-retry</artifactId>
+        </dependency>
```

Comment:
do we want to add another two dependencies due to this feature?

Reply:
I am able to do this by ugly loop, I just thought this solution is pretty and it can be used again in different cases.